api4/upload.go

```diff
@@ -16,9 +16,9 @@ import (
 )
 
 func (api *API) InitUpload() {
-	api.BaseRoutes.Uploads.Handle("", api.APISessionRequired(createUpload)).Methods("POST")
-	api.BaseRoutes.Upload.Handle("", api.APISessionRequired(getUpload)).Methods("GET")
-	api.BaseRoutes.Upload.Handle("", api.APISessionRequired(uploadData)).Methods("POST")
+	api.BaseRoutes.Uploads.Handle("", api.APISessionRequired(createUpload, model.ScopeFilesCreate)).Methods("POST")
+	api.BaseRoutes.Upload.Handle("", api.APISessionRequired(getUpload, model.ScopeFilesRead)).Methods("GET")
+	api.BaseRoutes.Upload.Handle("", api.APISessionRequired(uploadData, model.ScopeFilesUpdate)).Methods("POST")
```

Comment:
Wondering if this is essentially the same thing as `ScopeFilesCreate`. AFAIK you can't actually "update" a file after it's been saved

Reply:
1/5 It still is intention-revealing to have a separate scope.